scripts/gen_ed25519_key.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python
+"""Generate an ed25519 keypair, and store as base64-encoded text files.
+
+This script doesn't currently reuse the functions in `scriptworker.ed25519`, for
+easier standalone use. It could easily be a `console_script` though.
+
+"""
+from __future__ import print_function
+import base64
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives.asymmetric.ed25519 import Ed25519PrivateKey, Ed25519PublicKey
+import sys
+
+
+def private_key_from_string(key_str):
+    """Create an Ed25519PrivateKey from a base64-encoded string."""
+    return Ed25519PrivateKey.from_private_bytes(
+        base64.b64decode(key_str)
+    )
+
+
+def public_key_from_string(key_str):
+    """Create an Ed25519PublicKey from a base64-encoded string."""
+    return Ed25519PublicKey.from_public_bytes(
+        base64.b64decode(key_str)
+    )
+
+
+def b64_from_private_key(key):
+    """Get the base64 string from an Ed25519PrivateKey."""
+    return base64.b64encode(key.private_bytes(
+        encoding=serialization.Encoding.Raw,
+        format=serialization.PrivateFormat.Raw,
+        encryption_algorithm=serialization.NoEncryption()
+    ), None).decode('utf-8')
```

Comment:
I believe `None` is the default on `b64encode()` so it shouldn't be necessary to pass it. If I'm wrong about that, I think it'd be good to add a comment.

Reply:
That's right :) Fixed.